deploy/crds/tower.ansible.com_joblaunch_crd.yaml

```diff
@@ -31,6 +31,16 @@ spec:
                 type: string
               job_template_name:
                 type: string
+              inventory:
+                type: array
+                items:
+                  type: object
+              extra_vars:
+                type: object
+                target_clusters:
```

Comment:
This is a very specific extra vars element, is this pulled out before passing along? I would assume the target cluster would be covered with the link to the auth secret since that points at the correct host?

Reply:
@matburt   The target cluster field is reserved for passing our ACM managed cluster list to Ansible Tower. Hopefully the target cluster list can be used by some jobs.

Considering there may be more extra variables to be added. The extra_vars is designed to be a generic objects for holding various variable objects. The `target_cluster` is one of them, which is specific for our ACM usage